collectors/cluster_usage.go

```diff
@@ -106,29 +106,32 @@ func (c *ClusterUsageCollector) collect() error {
 		return err
 	}
 
-	tot, err := stats.Stats.TotalBytes.Float64()
+	var totBytes, usedBytes, availBytes, totObjects float64
+
+	totBytes, err = stats.Stats.TotalBytes.Float64()
 	if err != nil {
-		return err
+		log.Println("[ERROR] cannot extract total bytes:", err)
```

Comment:
`return` after error?

Also, might be worth making `infof()` and `errorf()` functions to save you the hassle of adding this prefix and such.